modules/service/src/test/scala/lucuma/odb/graphql/mutation/createObservation.scala

```diff
@@ -1274,7 +1292,11 @@ class createObservation extends OdbSuite {
             F2Reads.Reads_1,                // Default reads is 1 as por bright read mode
             None,                           // Explicit reads is Empty
             F2Reads.Reads_1,                // Default reads is 1 as por bright read mode
-            Some(F2Decker.LongSlit),        // Explicitly set
+            F2Decker.MOS,                   // Explicitly set
+            F2Decker.LongSlit,              // default to long slit
```

Comment:
Maybe we don't need to store this at all.  It seems to correspond 1:1 with the mode itself.  I mean, it can't be `Imaging` or `MOS` presumably and be a long slit observation?

Reply:
On the OCS it is derived from the observation type but can be overridden for engineering